openshift/README.md

```diff
@@ -10,6 +10,13 @@ To configure your developemnt deployment copy env-template.sh
 
 Update variables with your AWS,Openshift and Github credentials.
 
+### Generate RDS pasword
+
+To generate password you will require tool named pwgen.
+`pwgen -1cs`
```

Comment:
This generates fairly short password, we could use something like: `pwgen -1cs 32`